src/filesystem/impls/filer/lib/CSSRewriter.js

```diff
@@ -28,6 +28,7 @@ define(function (require, exports, module) {
         function aggregate(content, callback) {
             var urls = [];
             var urlRegex = new RegExp('url\\([\\\'\\"]?([^\\\'\\"\\)]+)[\\\'\\"]?\\)', 'g');
+            var urlQuotesRegex = new RegExp(/[\"\'](.*?\.css)[\"\']/, 'g');
```

Comment:
Yes indeed. I could've used that too :sweat_smile:. I'll change that bit.
Does the functioning look fine?